.github/workflows/pylint.yml

```diff
@@ -20,4 +20,4 @@ jobs:
         pip install pylint
     - name: Analysing the code with pylint
       run: |
-        pylint -d C0103 $(git ls-files '*.py')
\ No newline at end of file
+        pylint -d C0103 $(git ls-files '*.py' | grep -v 'hunter/')
```

Comment:
since we removed the hunter code, is this still needed?

Reply:
No, it is no longer needed.. I will remove the pipe.